tools/chrome-debug-extension/README.md

```diff
@@ -32,7 +32,7 @@ Since each web application logs telemetry in a different schema, you'll first be
 
 You can draft a configuration directly in the tool on the `Configuration Selection` page. You can always get back to that page by clicking the "gear" icon.
 
-To add your configuration to the dropdown list, you'll need to check in a change to `INSERT_PATH_TO_FILE` to add your configuration's URL to the list. Follow the instructions in that file.
+To add your configuration to the dropdown list, you'll need to check in a change to [`src/configuration/Configuration.types.ts`](https://github.com/microsoft/ApplicationInsights-JS/blob/a9c3948db6d7dddf58cc6450735958667a79d0ac/tools/chrome-debug-extension/src/configuration/Configuration.types.ts#L16) to add your configuration's URL to the list. Follow the instructions in that file.
```

Comment:
We will eventually need to update this to point to the master version

Reply:
updated to relative path